src/webgl/framebuffer.js

```diff
@@ -8,6 +8,24 @@ import {getTypedArrayFromGLType, getGLTypeFromTypedArray} from '../utils/typed-a
 import {log} from '../utils';
 import assert from 'assert';
 
+// Local constants - will collapse during minification
+const GL_FRAMEBUFFER = 0x8D40;
```

Comment:
Why we decide to have these GL constants defined here instead of extracting them from our "GL" struct?